src/Realm.C

```diff
@@ -3708,7 +3708,7 @@ Realm::check_job(bool get_node_count)
   // set number of nodes, check job run size
   if (get_node_count) {
     size_t localNodeCount =
-      ioBroker_->get_input_io_region()->get_property("node_count").get_int();
```

Comment:
Yep, sorry about that change. (That was an artifact of our conversion from teuchos::rcp to std::shared_ptr.)